include/user_settings.h

```diff
@@ -122,9 +126,7 @@
 #endif
 
 /* Disables - For minimum wolfCrypt build */
-#define NO_AES
```

Comment:
Use #ifndef WOLFTPM instead of removing the line

Reply:
good point